corfudb-tools/README

```diff
@@ -0,0 +1,33 @@
+UFO Browser Editor Tool
+
+The Browser and Editor tools are used for inspecting and troubleshooting UFO
+data inside Corfu.
+
+The tool is schemaless, i.e., it does not need to be packaged with protobuf
+definitions or proto files to serialize/deserialize the data.
+
+--------
+Browser
+--------
+The browser supports read operation which reads and outputs all entries in a
+given table and namespace.
+We plan to add a user script to invoke the browser but as of now, it can be used
+with the following command-
+
+java -cp <corfu-tools jar>
```

Comment:
I think the example should be invoked with actual arguments:
`java -cp corfudb-tools-0.3.0-SNAPSHOT-shaded.jar org.corfudb.browser.CorfuBrowserMain`

Since docopt generates the usage with --help, I don't think we also need to document them here

Reply:
done